docs/source/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import sphinx_rtd_theme
+from pyogrio import __version__
+
+
+autodoc_mock_imports = [
+    "geopandas",
+    "pygeos",
+]
+
+# -- Project information -----------------------------------------------------
+
+project = "pyogrio"
+copyright = "2021, Brendan C. Ward"
+author = "Brendan C. Ward"
```

Comment:
Can this be left out?  I don't see that this is used in Geopandas' conf.py, so we probably don't need it here.  Otherwise could be expanded to `Brendan C. Ward and pyogrio contributors`